src/components/OptionsTiles/OptionTiles.Overview.stories.mdx

```diff
@@ -0,0 +1,530 @@
+      const options = [
+        {
+          id: 'chocolate_error',
+          value: 'chocolate',
+          label: 'Chocolate',
+        },
+        {
+          id: 'strawberry_error',
+          value: 'strawberry',
+          label: 'Strawberry',
+        },
+        {
+          id: 'vanilla_error',
+          value: 'vanilla',
+          label: 'Vanilla',
+          disabled: true,
+        },
+      ];
+      return (
+        <OptionTiles
+          name="required"
+          title="Ice cream flavors"
+          description="Only if you eat all your dinner"
+          isRequired
+          value={value}
+          onChange={event => setValue(event.target.value)}
+          options={options}
+          error="You should really select an ice cream flavor!"
+        />
+      );
+    }}
+  </Story>
+</Canvas>
+
+
+## Full Width (or lack thereof)
+
+The OptionsTile component has fullWidth (100%) enabled by default but it can be disabled if need be.
+
+<Canvas isExpanded>
+  <Story name="Full Width">
+    {() => {
+      const [valueHorizontal, setValueHorizontal] = useState([]);
+      const [valueVertical, setValueVertical] = useState([]);
+      const optionsVertical = [
+        {
+          id: 'chocolate_fullwidth_vertical',
+          value: 'chocolate',
+          label: 'Chocolate',
+        },
+        {
+          id: 'strawberry_fullwidth_vertical',
+          value: 'strawberry',
+          label: 'Strawberry',
+        },
+        {
+          id: 'vanilla_fullwidth_vertical',
+          value: 'vanilla',
+          label: 'Vanilla',
+        },
+      ];
+      const optionsHorizontal = [
+        {
+          id: 'chocolate_fullwidth_horizontal',
+          value: 'chocolate',
+          label: 'Chocolate',
+        },
+        {
+          id: 'strawberry_fullwidth_horizontal',
+          value: 'strawberry',
+          label: 'Strawberry',
+        },
+        {
+          id: 'vanilla_fullwidth_horizontal',
+          value: 'vanilla',
+          label: 'Vanilla',
+        },
+      ];
+      return (
+        <>
+          <p className="m-bottom-md font-size-lg font-weight-bold">Vertical</p>
+          <OptionTiles
+            name="fullWidthVertical"
+            value={valueVertical}
+            onChange={event => setValueVertical(event.target.value)}
+            options={optionsVertical}
+            direction="column"
+            isFullWidth={false}
+          />
+          <p className="m-v-md font-size-lg font-weight-bold">Horizontal</p>
+          <OptionTiles
+            name="fullWidthHorizontal"
+            value={valueHorizontal}
+            onChange={event => setValueHorizontal(event.target.value)}
+            options={optionsHorizontal}
+            direction="row"
+            isFullWidth={false}
+          />
+        </>
+      );
+    }}
+  </Story>
+</Canvas>
+
+## Custom Content
+
+The tiles can render custom content with a `render` function passed in the option object.
```

Comment:
Why not use the `children` prop?

Reply:
Hey @RyanMacBern let's chat a bit more about this if you have a chance. Part of this was trying to roughly stick to the existing api in the app component which rendered the options using the options prop, rather than children, and so  the compnent as a whole is a self-closing one.

IN general we prefer subcomponents with children, but not sure what we feel is best in this case.

Reply:
We chatted offline. This makes sense in this case.